loom/script/reflection/lsAssembly.h

```diff
@@ -67,12 +67,7 @@ class Assembly {
     // the assemblies we reference
     utArray<utString> references;
 
-    Type **ordinalTypes;
-
-    /*
-     * Closes the assembly, cleaning up associated hashes and Types
-     */
-    void close();
+    utArray<Type*> *ordinalTypes;
```

Comment:
I changed it to utArray since lmNew/Delete do not support new[] / delete[] and utArray is suggested. If not compiled in debug mode, I don't think this change has a negative impact, since allocation is done in a one-time `resize()` and lookup is done via `at()`, which is defined as follows:
```
UT_INLINE T& at(UTsize idx) { UT_ASSERT(idx >= 0 && idx < m_capacity); return m_data[idx]; }
UT_INLINE const T& at(UTsize idx) const { UT_ASSERT(idx >= 0 && idx < m_capacity); return m_data[idx]; }
```

Where `UT_INLINE` is defined per platform to be `inline`, while `UT_ASSERT` compiles away if there aren't any `DEBUG` and similar directives defined. So as far as I can see, this shouldn't affect the system in a big way.